src/api/java/org/commcare/api/persistence/UserSqlSandbox.java

```diff
@@ -53,6 +62,32 @@ public UserSqlSandbox(String username) {
     }
 
     @Override
+    public IStorageUtilityIndexed<StorageIndexedTreeElementModel> getIndexedFixtureStorage(String fixtureName) {
+        String tableName = StorageIndexedTreeElementModel.getTableName(fixtureName);
+        return new SqliteIndexedStorageUtility<>(StorageIndexedTreeElementModel.class, username, tableName, path);
+    }
+
+    @Override
+    public void setupIndexedFixtureStorage(String fixtureName,
+                                           StorageIndexedTreeElementModel exampleEntry,
+                                           Set<String> indices) {
+        // TODO PLM: delete table if it already exists
+        // TODO PLM: create indexes over table
+        String tableName = StorageIndexedTreeElementModel.getTableName(fixtureName);
+        new SqliteIndexedStorageUtility<>(exampleEntry, username, tableName, path);
+    }
+
+    @Override
+    public Pair<String, String> getIndexedFixturePathBases(String fixtureName) {
+        throw new RuntimeException("implement in similar fashion as AndroidSandbox implementation");
```

Comment:
I was hoping to defer these implementations to @wpride because I'm not sure how this (formplayer specific?) DB setup works. I'll sync up with Will about this tomorrow.